### PR TITLE
[Lib-464] Custom QuerySets for models

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: src/querySet.js [hunk 1]
```diff
@@ -513,4 +513,13 @@ const QuerySet = stampit.compose(
   Raw
 );
 
+export const BaseQuerySet = stampit.compose(
```
Comment (dancio) [inline src/querySet.js:516]: just `QuerySet` would be fine :P but with `Base` works too

File: test/integration/apiKeysTest.js [hunk 2]
```diff
@@ -3,17 +3,22 @@ import Promise from 'bluebird';
 import _ from 'lodash';
 import Syncano from '../../src/syncano';
 import {ValidationError} from '../../src/errors';
-import {suffix, credentials} from './utils';
+import {suffix, credentials, createCleaner} from './utils';
 
-describe('ApiKey', function() {
+describe.only('ApiKey', function() {
```
Comment (dancio) [inline test/integration/apiKeysTest.js]: `only`

**Comment by dancio** [comment]
besides minor things looks good

File: test/integration/apiKeysTest.js (hunk 2 above)
Comment (jakubbilko) [inline test/integration/apiKeysTest.js]: wrrrr

File: src/querySet.js (hunk 1 above)
Comment (jakubbilko) [inline src/querySet.js:516]: But we're already exporting a `QuerySet` const from this module

Comment (dancio) [inline src/querySet.js:516]: oh, my bad :)

**Pull request merged**